src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -452,6 +452,9 @@ public object[] Geometry()
                     if (solid != null)
                     {
                         var geomObjs = solid.ConvertToMany();
```

Comment:
curious if this fails, does it leave behind any ASM geometry in memory which is not disposed?